pytest.ini

```diff
@@ -7,4 +7,4 @@ testpaths = tests
 md_report = True
 
 # https://pypi.org/project/pytest-rerunfailures/
-reruns = 3
+reruns = 1
```

Comment:
If you want to remove it, then set it to `reruns = 0`.

`1` means it will rerun once.

Reply:
So confusing! I understand "to re-run all test failures, use the --reruns command line option with the maximum number of times you’d like the tests to run" as "0" means don't even try it.

Reply:
We also can remove the tool for now.

Reply:
Actually, current revision removes `pytest-rerunfailures` completely. We can always roll it again if it is required.

Reply:
OK, fine as well